migrate/20231116_project_flag.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+Sequel.migration do
+  up do
+    alter_table(:project) do
+      add_column :feature_flags, :jsonb
+    end
+
+    run "UPDATE project SET feature_flags = '{}'"
+
+    alter_table(:project) do
+      set_column_not_null :feature_flags
+      set_column_default :feature_flags, "{}"
+    end
```

Comment:
```suggestion
    alter_table(:project) do
      add_column :feature_flags, :jsonb, null: false, default: "{}"
    end
```

Reply:
what about just using semaphores?

Reply:
I initially did this, but rubocop didn't like it.

Reply:
Hmm, it didn't complain this time. It seems if you don't add `null:false` it errors out with this;

```
Sequel/ColumnDefault: Don't create new column with default values
      add_column :feature_flags, :jsonb, default: "{}"
```

But if you add both null:false and default value, it just passes.

Reply:
Yeah I talked to the maintainer about that one, it's to avoid table rewriting, let's just disable the lint globally.

Reply:
#885